base_delivery_carrier_label/views/delivery.xml

```diff
@@ -52,7 +53,8 @@
     <field name="model">delivery.carrier.option</field>
     <field name="type">tree</field>
     <field name="arch" type="xml">
-      <tree string="delivery_carrier_option">
+      <tree string="delivery_carrier_option" decoration-warning="not active">
```

Comment:
"decoration-muted" should be more appropriate for not active flag ?

Reply:
thanks, that's better indeed